engine/language-server/src/main/scala/org/enso/languageserver/protocol/json/JsonConnectionController.scala

```diff
@@ -386,18 +386,16 @@ class JsonConnectionController(
       )
 
     case ContextRegistryProtocol.VisualizationEvaluationFailed(
-          contextId,
-          visualizationId,
-          expressionId,
+          ctx,
           message,
           diagnostic
         ) =>
       webActor ! Notification(
         VisualizationEvaluationFailed,
         VisualizationEvaluationFailed.Params(
-          contextId,
-          visualizationId,
-          expressionId,
+          ctx.contextId,
```

Comment:
Rather than sending the `VisualizationContext` just left it as-is, so that we don't break the protocol.